rust/platform/triple_mappings.bzl

```diff
@@ -61,6 +62,8 @@ _CPU_ARCH_TO_BUILTIN_PLAT_SUFFIX = {
     "powerpc64le": None,
     "riscv32": "riscv32",
     "riscv32imc": "riscv32",
+    "riscv64": "riscv64",
+    "riscv64gc": "riscv64",
```

Comment:
I don't see a constraint for this defined at https://github.com/bazelbuild/platforms/blob/0.0.6/cpu/BUILD. Won't this cause problems if anyone tries to match it?

Reply:
oh nevermind, they get mapped to `riscv54`. No issue here.